File: ProjectEuler2/ProjectEuler2.py
def ProjectEuler2():
	s = 2
	first = 1
	second = 2
	nex = first + second
	while nex < 4000000:
		if nex % 2 == 0:
			s += nex
		first = second
		second = nex
		nex = first + second
	return s

print(ProjectEuler2())

File: ProjectEuler2/ProjectEuler2.java
public class ProjectEuler2 {
	public static void main(String[] args){
		int sum = 2;
		int first = 1, second = 2;
		int next = first + second;
		while (next < 4000000){
			if (next % 2 == 0){
				sum += next;
			}
			first = second;
			second = next;
			next = first + second;
		}
		System.out.println(sum);
	}
}